### ISSUE TITLE
Fatal error in Dynamic Tags pods-url.php

### ISSUE THREAD

**Issue by lucaskeiser** [elementor/elementor#8396]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description
There is a single post template. When I went to edit it, this error popped up.

It looks like the issue might be due to how the function 'pods' handles pods that do not exist. I do not see any existence check before the code starts calling the fields property.

I am thinking a pod field might have been deleted after the template was built.

[https://pods.io/docs/code/pods/](https://pods.io/docs/code/pods/)

`Notice: Trying to get property 'fields' of non-object in /home/bnefm0j7s6ph/public_html/staging/assets/extensions/elementor-pro/modules/dynamic-tags/pods/tags/pods-url.php on line 46

Fatal error: Uncaught Error: Call to a member function field() on bool in /home/bnefm0j7s6ph/public_html/staging/assets/extensions/elementor-pro/modules/dynamic-tags/pods/tags/pods-url.php:47 Stack trace: #0 /home/bnefm0j7s6ph/public_html/staging/assets/extensions/elementor/core/dynamic-tags/data-tag.php(44): ElementorPro\Modules\DynamicTags\Pods\Tags\Pods_URL->get_value(Array) #1 /home/bnefm0j7s6ph/public_html/staging/assets/extensions/elementor/core/dynamic-tags/manager.php(207): Elementor\Core\DynamicTags\Data_Tag->get_content() #2 [internal function]: Elementor\Core\DynamicTags\Manager->get_tag_data_content('4f28596', 'Pods-url', Array) #3 /home/bnefm0j7s6ph/public_html/staging/assets/extensions/elementor/core/dynamic-tags/manager.php(100): call_user_func_array(Array, Array) #4 /home/bnefm0j7s6ph/public_html/staging/assets/extensions/elementor/core/dynamic-tags/manager.php(65): Elementor\Core\DynamicTags\Manager->parse_tag_text('[elementor-tag ...', Array, Array) #5 [internal function]: Elementor\Core\DynamicTags\Mana in /home/bnefm0j7s6ph/public_html/staging/assets/extensions/elementor-pro/modules/dynamic-tags/pods/tags/pods-url.php on line 47`

## Steps to reproduce

From best I can tell, there was a single template that is pulling data using something from the Pods module in Elementor.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [ ] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by shilo-ey** [comment]
Thanks for reporting.

A patch will be released soon and address this issue.

Follow this thread for more updates.

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor Pro v2.5.11

Feel free to update

Thanks!
